scripts/github-build-info.mjs

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const buildNumber = [ 'GITHUB_RUN_NUMBER', 'GITHUB_RUN_ATTEMPT', 'GITHUB_RUN_ID']
```

Comment:
Is it a standard order of build number components? I'm thinking it might be better to make the run attempt last

Reply:
oh actually just realized that run ID will probably be unique for every attempt so it won't make it any more consistent. Nevermind